guides/micronaut-liquibase/micronaut-liquibase.adoc

```diff
@@ -0,0 +1,193 @@
+include::{commondir}/common-header.adoc[]
+
+You are going to use https://docs.oracle.com/en/java/javase/16/language/records.html[Record Classes] in a Micronaut Application.
+
+____
+Record classes, which are a special kind of class, help to model plain data aggregates with less ceremony than normal classes.
+
+A record declaration specifies in a header a description of its contents; the appropriate accessors, constructor, equals, hashCode, and toString methods are created automatically. A record's fields are final because the class is intended to serve as a simple "data carrier".
+____
+
+include::{commondir}/common-requirements.adoc[]
+
+include::{commondir}/common-completesolution.adoc[]
+
+include::{commondir}/common-cli-or-launch.adoc[]
+
+[source,bash]
+----
+mn create-app
+   example.micronaut.micronautguide \
+   --features=data-jdbc,postgres,liquibase \ 
+   --build=@build@ \
+   --lang=@lang@ \
+----
+
+
+NOTE: To use Java records, specify JDK 17 when you create the application
+
+include::{commondir}/common-build-lang-arguments.adoc[]
+
+include::{commondir}/common-default-package.adoc[]
+
+NOTE: If you use https://launch.micronaut.io[Micronaut Launch], select "Micronaut Application" as application type and add the `postgres`, `data-jdbc` and `liquibase` as features.
+
+include::{commondir}/common-annotationprocessors.adoc[]
+
+=== Create Entity
+
+Create a `@MappedEntity` to save persons. Initially, consider name and age required. Use the `int` primitive for the age.
+
+source:Person[tags=clazz|ageint|gettersetters]
+
+callout:mapped-entity[1]
+callout:mapped-entity-id[2]
+callout:generated-value[3]
+callout:version-optimistic-locking[4]
+
+=== Database Migration with Liquibase
+
+We need a way to create the database schema. For that, we use
+https://micronaut-projects.github.io/micronaut-liquibase/latest/guide/[Micronaut integration with Liquibase].
+
+Add the following snippet to include the necessary dependencies:
+
+dependency:micronaut-liquibase[groupId=io.micronaut.liquibase]
+
+Configure the database migrations directory for http://www.liquibase.org[Liquibase] in `application.yml`.
+
+resource:application.yml[tag=liquibase]
+
+Create the following files with the database schema creation:
+
+resource:db/liquibase-changelog.xml[]
+
+resource:db/changelog/01-create-person.xml[]
```

Comment:
Please indent the files with four spaces instead of eight because it's more complicated to read due to the horizontal scroll:  

![image](https://user-images.githubusercontent.com/559192/142638804-2d110eda-268d-4b16-aff7-f26f530cd9db.png)